src/audiomixerboard.cpp

```diff
@@ -119,6 +128,10 @@ CChannelFader::CChannelFader ( QWidget*     pNW,
     pFader->setAccessibleName ( tr ( "Local mix level setting of the current audio "
         "channel at the server" ) );
 
+    pPan->setWhatsThis ( tr ( "<b>Panning:</b> Sets the panning position from Left to Right of the channel. "
```

Comment:
I have changed this recently. I do not want to have the HTML codes in the translation (see, e.g, the "Mute" help text as a reference).